src/styles/app.scss

```diff
@@ -3,15 +3,17 @@
 .app {
   padding: 0 30px;
   max-width: 500px;
-  height: 500px; 
-  margin: 0 auto; 
+  height: 500px;
+  margin: 0 auto;
   font-family: $font-family;
   background-color: $primary;
-  display: flex; 
-  justify-content: center; 
-  align-items: center; 
+  display: flex;
+  justify-content: center;
+  align-items: center;
   flex-direction: column;
-  box-shadow: 0 0 0 10px $secondary;
+  // this is just a border
```

Comment:
I did this because the border wasn't looking as I expected at the time, but a box shadow was. it could be different now